src/qt/bitcoin.cpp

```diff
@@ -457,11 +460,11 @@ int GuiMain(int argc, char* argv[])
 
     /// 2. Parse command-line options. We do this after qt in order to show an error if there are problems parsing these
     // Command-line options take precedence:
-    node->setupServerArgs();
+    SetupServerArgs(*node->context());
```

Comment:
commit 427c8da7113527cf3809cb568cd3fe529e56b259: The documentation for context() says "Useful for testing". Did you mean to write `node_context` instead?
```suggestion
    SetupServerArgs(node_context);
```

Reply:
re: https://github.com/bitcoin-core/gui/pull/35/files#r471397426

> commit [427c8da](https://github.com/bitcoin-core/gui/commit/427c8da7113527cf3809cb568cd3fe529e56b259): The documentation for context() says "Useful for testing". Did you mean to write `node_context` instead?

Changed, agree this is a little better. SetupServerArgs should really not be using NodeContext at all (argument is removed in https://github.com/bitcoin/bitcoin/pull/10102), but for this PR which is not in the GUI repository, I wanted to avoid changing non-gui code.